gvr-3dcursor-simple/app/src/main/java/org/gearvrf/io/cursorsimple/CursorMain.java

```diff
@@ -63,10 +64,25 @@ public void onInit(GVRContext gvrContext) {
         mainScene = gvrContext.getNextMainScene();
         mainScene.getMainCameraRig().getLeftCamera().setBackgroundColor(Color.BLACK);
         mainScene.getMainCameraRig().getRightCamera().setBackgroundColor(Color.BLACK);
-        cursorManager = new CursorManager(gvrContext, mainScene);
+        List<IoDevice> devices = new ArrayList<IoDevice>();
+
+        //_VENDOR_TODO_ register the devices with Cursor Manager here.
+        /*
+        TemplateDevice device1 = new TemplateDevice(gvrContext, "template_1", "Right controller");
+        TemplateDevice device2 = new TemplateDevice(gvrContext, "template_2", "Left controller");
+        devices.add(device1);
+        devices.add(device2);
+        */
+
+        /*
+        HandTemplateDevice device = new HandTemplateDevice(gvrContext, mainScene);
+        devices.addAll(device.getDeviceList());
```

Comment:
Can add HandTemplateDevice device, and device1, and device2 as member variables. So trying out the template device will only involve un-commenting code. Right now when you uncomment device.close(). It does not recognize device as its not a member variable.